src/components/UserDetailSections/ContactInfo/ContactInfo.js

```diff
@@ -56,6 +56,14 @@ const ContactInfo = ({
           />
         </Col>
       </Row>
+      <Row>
+        <Col xs={12}>
+          <KeyValue
+            label={<FormattedMessage id="ui-users.contact.preferredEmailCommunication" />}
+            value={_.get(user, ['preferredEmailCommunication'], '-').join(', ')}
```

Comment:
Could you please clarify can we move this to function?

Reply:
Could you please clarify can we use `import { NoValue } from '@folio/stripes/components';` instead of '-'?